configs/13.0/deb/monolithic/rules

```diff
@@ -38,7 +38,7 @@ ifeq (__USE_SYSTEMD__, yes)
 endif
 
 # Copy the packages' files.
-	dh_install --fail-missing
+	dh_missing --fail-missing
```

Comment:
I'm certainly not an expert, but this doesn't seem right.

The comment before the current `dh_install` says "Copy the packages' files.".  My understanding is that is the purpose of `dh_install`.  And `dh_install --fail-missing` will _also_ check for files which were not installed, but presumably should be.  The use of `dh_missing --fail-missing` is just to perform that latter step, correct?  Shouldn't the code be changed to:
```
        dh_install
        dh_missing --fail-missing
```
?

Reply:
@ThinkOpenly you're right. I updated the patch.